pkg/question/question.go

```diff
@@ -618,64 +649,67 @@ func AskIamProfile(i *iamhelper.IAMHelper, defaultIamProfile string) (string, er
 		}
 	}
 
-	defaultOptionRepr, defaultOptionValue := "Do not attach IAM profile", cli.ResponseNo
+	defaultOptionValue := cli.ResponseNo
 	noOptionRepr, noOptionValue := "Do not attach IAM profile", cli.ResponseNo
 
 	data := [][]string{}
 	indexedOptions := []string{}
-	var optionsText string
 	if len(instanceProfiles) > 0 {
 		counter := 0
 		for _, profile := range instanceProfiles {
 			indexedOptions = append(indexedOptions, *profile.InstanceProfileName)
-			data = append(data, []string{fmt.Sprintf("%d.", counter+1), *profile.InstanceProfileName, *profile.InstanceProfileId,
-				profile.CreateDate.String()})
+			data = append(data, []string{*profile.InstanceProfileName, *profile.InstanceProfileId, profile.CreateDate.String()})
 			if defaultIamProfile == *profile.InstanceProfileName {
-				defaultOptionRepr, defaultOptionValue = *profile.InstanceProfileName, *profile.InstanceProfileName
+				defaultOptionValue = *profile.InstanceProfileName
 			}
 			counter++
 		}
-	} else {
-		optionsText = "No IAM Profiles available\n"
 	}
 
 	// Add the do not attach IAM profile option at the end
 	indexedOptions = append(indexedOptions, noOptionValue)
-	data = append(data, []string{fmt.Sprintf("%d.", len(data)+1), noOptionRepr, "", ""})
-	optionsText = table.BuildTable(data, []string{"Option", "PROFILE NAME", "PROFILE ID",
-		"Creation Date"})
+	data = append(data, []string{noOptionRepr})
 
 	question := "IAM Profile"
-	answer := AskQuestion(&AskQuestionInput{
-		QuestionString:    question,
-		DefaultOptionRepr: &defaultOptionRepr,
-		DefaultOption:     &defaultOptionValue,
-		OptionsString:     &optionsText,
-		IndexedOptions:    indexedOptions,
+	headers := []string{"PROFILE NAME", "PROFILE ID", "Creation Date"}
```

Comment:
The different casing styles in these table headers is odd. Perhaps we should standardize on initial capitals ("Profile Name") and then let the table rendering code decide whether to keep that capitalization or make everything uppercase. That way we can change the style all in one place.

Reply:
When the headers are created theyre styled with the Bold styling from Lipgloss. This makes them capitalized and bolded.